### ISSUE TITLE
Message react tool tray fails to render emoji's when Button Shapes Accessibility option is enabled 

### ISSUE THREAD

**Issue by Doty1154** [element-hq/element-x-ios#2406]
### Steps to reproduce

I am going blind and have accessibility options enabled on my phone. Specifically:
- Button Shapes
- On/Off Labels
- Reduce Transparency
- Increase Contrast
- Differentiate without color
- And my text scale is turned up all of the way but before "Larger Accessibility Sizes" is enabled
From any channel or dm chat. When i initiate a message react for a message or picture i hope to see & use the emoji's that i react to a message photo. 

### Outcome

The emoji widget menu fails to load and shows blank empty gray boxes where emoji's should load. 
Once i disable Button shapes the emoji's load but are clipping into one another

### Your phone model

iPhone 11 Pro Max

### Operating system version

17.3

### Application version

1.5.4

### Homeserver

catboyindustries.co Synapse 1.99.0

### Will you send logs?

Yes

**Comment by Doty1154** [comment]
![IMG_0020](https://github.com/element-hq/element-x-ios/assets/7087748/afdbfa89-68a8-4b6d-82f3-8cadba2f5904)

**Comment by Doty1154** [comment]
[console.log](https://github.com/element-hq/element-x-ios/files/14100896/console.log)

**Comment by Velin92** [comment]
A PR has been opened to fix this issue:
https://github.com/element-hq/element-x-ios/pull/2780
